cloudprober/templates/deployment.yaml

```
@@ -12,6 +12,9 @@ spec:
   template:
     metadata:
       annotations:
+        {{- if $.Values.podAnnotations }}
+          {{ toYaml .Values.podAnnotations | nindent 8 }}
```

Minor change to kill any spaces in the beginning.

```suggestion
          {{- toYaml .Values.podAnnotations | nindent 8 }}
```

---

Sure yes suggestion committed 👍